src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpResponseMessageHelper.cs

```diff
@@ -27,7 +27,9 @@ internal class HttpResponseMessageHelper
         public HttpResponseMessageHelper(HttpResponseMessage httpResponseMessage, HttpChannelFactory<IRequestChannel> factory)
         {
             Contract.Assert(httpResponseMessage != null);
-            Contract.Assert(httpResponseMessage.RequestMessage != null);
+            // httpResponseMessage.RequestMessage is null when running in browser ( WebAssembly )
+            if (!Fx.IsWasm)
```

Comment:
This is a bug in WebAssembly. I would prefer that the WebAssembly implementation is fixed here.

Reply:
I agree it would be better if it was fixed in mono. 
This is just a quick fix which worked for me.